packages/api-server/src/methods/modules/eth.ts

```diff
@@ -814,14 +816,23 @@ export class Eth {
   }
 
   async sendRawTransaction(args: [string], callback: Callback) {
-    const data = args[0];
-    const rawTx = await generateRawTransaction(data, this.rpc);
-    const moleculeTx = new Reader(
-      schemas.SerializeL2Transaction(types.NormalizeL2Transaction(rawTx))
-    ).serializeJson();
-    const result = await this.rpc.submit_l2transaction(moleculeTx);
-    console.log('sendRawTransaction hash:', result);
-    callback(null, result);
+    try {
+      const data = args[0];
+      const rawTx = await generateRawTransaction(data, this.rpc);
+      const moleculeTx = new Reader(
+        schemas.SerializeL2Transaction(types.NormalizeL2Transaction(rawTx))
+      ).serializeJson();
+      const result = await this.rpc.submit_l2transaction(moleculeTx);
+      console.log('sendRawTransaction hash:', result);
+      callback(null, result);
+    } catch (error) {
+      console.error(error);
+      callback({
+        // https://www.jsonrpc.org/specification | 5.1 Error object
+        code: INTERNAL_ERROR,
```

Comment:
This is not an internal error, users may submit invalid data, and these behaviors are in our expectation, IMO we should return an `INVALID_PARAMS` error code.

Reply:
hupeng has designed a [middleware for parameters validation,](https://github.com/bitrocks/godwoken-web3/blob/4c6f5e3746b39bd4d8f5181c3ed906164aeb7b45/packages/api-server/src/methods/validator.ts#L14-L48)
handling `INVALID_PARAMS` errors.

But this sendRawTransaction method was not using the parameters validation. 
I am not sure why we didn't use it, so I choose `INTERNAL_ERROR` error temporally and make [a little optimization](https://github.com/nervosnetwork/godwoken-web3/pull/4/commits/3fe01b0ade6f77eaf5bc8901c3ba075657e625a9).